src/dotnet/APIView/APIViewWeb/Repositories/CommentsManager.cs

```diff
@@ -31,12 +40,21 @@ public async Task<ReviewCommentsModel> GetReviewCommentsAsync(string reviewId)
             return new ReviewCommentsModel(reviewId, comments);
         }
 
-        public async Task AddCommentAsync(ClaimsPrincipal user, CommentModel comment)
+        public async Task AddCommentAsync(ClaimsPrincipal user, CommentModel comment, string reviewId)
         {
             comment.Username = user.GetGitHubLogin();
             comment.TimeStamp = DateTime.Now;
 
             await _commentsRepository.UpsertCommentAsync(comment);
+            await SubscribeAndNotify(user, comment, reviewId);
+        }
+
+        public async Task SubscribeAndNotify(ClaimsPrincipal user, CommentModel comment, string reviewId)
```

Comment:
Feels like this belongs to ReviewManager or notification manager.

Reply:
Moved to notification manager.